### PR TITLE
docs: fix zh punctuations

### PR DESCRIPTION
**Pull request by zmstone**
Fixes <issue-number>

**If your build fails** due to your commit message not passing the build checks, please review the guidelines here: https://github.com/emqx/emqx/blob/master/CONTRIBUTING.md.

## PR Checklist
Please convert it to a draft if any of the following conditions are not met. Reviewers may skip over until all the items are checked:

- [x] Added tests for the changes
- [x] Changed lines covered in coverage report
- [x] Change log has been added to `changes/` dir
- [x] For EMQX 4.x: `appup` files updated (execute `scripts/update-appup.sh emqx`)
- [x] For internal contributor: there is a jira ticket to track this change, and another jira tickt to track doc updates (if any)
- [x] In case of non-backward compatible changes, reviewer should check this item as a write-off, and add details in **Backward Compatibility** section

### CODE REVIEW

**Review by ieQu1 — APPROVED** [review]

File: apps/emqx_authz/i18n/emqx_authz_api_cache_i18n.conf [hunk 1]
```diff
@@ -2,7 +2,7 @@ emqx_authz_api_cache {
   authorization_cache_delete {
     desc {
       en: """Clean all authorization cache in the cluster."""
-      zh: """清除集群中所有鉴权数据缓存"""
+      zh: """清除集群中所有授权数据缓存"""
```
Comment (HJianBo) [inline apps/emqx_authz/i18n/emqx_authz_api_cache_i18n.conf:5]:
```suggestion
      zh: """清除集群中所有授权数据缓存。"""
```

File: apps/emqx_authz/i18n/emqx_authz_api_schema_i18n.conf [hunk 2]
```diff
@@ -26,7 +26,7 @@ emqx_authz_api_schema {
   rules {
     desc {
       en: """Authorization static file rules."""
-      zh: """静态鉴权文件规则"""
+      zh: """静态授权文件规则"""
```
Comment (HJianBo) [inline apps/emqx_authz/i18n/emqx_authz_api_schema_i18n.conf:29]:
```suggestion
      zh: """静态授权文件规则。"""
```

File: apps/emqx_authz/i18n/emqx_authz_api_schema_i18n.conf [hunk 3]
```diff
@@ -111,7 +111,7 @@ emqx_authz_api_schema {
   collection {
     desc {
       en: """`MongoDB` collection containing the authorization data."""
-      zh: """`MongoDB` 鉴权数据集"""
+      zh: """`MongoDB` 授权数据集"""
```
Comment (HJianBo) [inline apps/emqx_authz/i18n/emqx_authz_api_schema_i18n.conf:114]:
```suggestion
      zh: """`MongoDB` 授权数据集。"""
```

File: apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf [hunk 4]
```diff
@@ -83,7 +83,7 @@ It is NOT allowed to configure two or more sources of the same type.
   file {
     desc {
       en: """Authorization using a static file."""
-      zh: """使用静态文件鉴权"""
+      zh: """使用静态文件授权"""
```
Comment (HJianBo) [inline apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf:86]:
```suggestion
      zh: """使用静态文件授权。"""
```

File: apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf [hunk 5]
```diff
@@ -213,7 +213,7 @@ and the old file will not be used anymore.
   mnesia {
     desc {
       en: """Authorization using a built-in database (mnesia)."""
-      zh: """使用内部数据库鉴权 (mnesia)."""
+      zh: """使用内部数据库授权（mnesia）。"""
```
Comment (qzhuyan) [inline apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf:216]: nit, may remove space before' 。'

Comment (qzhuyan) [reply apps/emqx_authz/i18n/emqx_authz_schema_i18n.conf:216]: oh, sorry it is due to `)`

**Review by qzhuyan — APPROVED** [review]

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 6]
```diff
@@ -1541,7 +1489,8 @@ Enable TLS session reuse.
 common_ssl_opts_schema_depth {
     desc {
         en: """
-Maximum number of non-self-issued intermediate certificates that can follow the peer certificate in a valid certification path. So, if depth is 0 the PEER must be signed by the trusted ROOT-CA directly; if 1 the path can be PEER, CA, ROOT-CA; if 2 the path can be PEER, CA, CA, ROOT-CA, and so on. The default value is 10.
+Maximum number of non-self-issued intermediate certificates that can follow the peer certificate in a valid certification path.
+So, if depth is 0 the PEER must be signed by the trusted ROOT-CA directly; if 1 the path can be PEER, CA, ROOT-CA; if 2 the path can be PEER, CA, CA, ROOT-CA, and so on. The default value is 10.
```
Comment (qzhuyan) [inline apps/emqx/i18n/emqx_schema_i18n.conf:1493]: for 
>  if 1 the path can be PEER, CA, ROOT-CA;

maybe replace `CA` with `intermediate CA` ?

Comment (zmstone) [reply apps/emqx/i18n/emqx_schema_i18n.conf:1493]: will fix in the next PR.

File: apps/emqx/i18n/emqx_schema_i18n.conf [hunk 7]
```diff
@@ -2398,7 +2301,7 @@ fields_tcp_opts_recbuf {
 The TCP receive buffer (OS kernel) for the connections.
 """
         zh: """
-连接的 TCP 接收缓冲区（OS内核）。
+连接的 TCP 接收缓冲区（OS 内核）。
```
[the remark this answers is not in the document]
Comment (zmstone) [reply apps/emqx/i18n/emqx_schema_i18n.conf:2304]: will fix in the next pr

**Pull request merged**